features/openhab-addons-external/src/main/resources/conf/iec6205621meter.cfg

```diff
@@ -0,0 +1,19 @@
+#############################  IEC 620562-21 Meter Binding ############################
+#
+# the serial port to use for connecting to the metering device e.g. COM1 for Windows
+# and /dev/ttyS0 or /dev/ttyUSB0 for Linux
+#<meter-id1>.serialPort=/dev/ttyS0
+meter1.serialPort=/dev/ttyS0
```

Comment:
This .cfg file should be installed with all properties commented out (with a single `#` before the property and not followed by a space.).  The point is that the file is installed as an example to follow, not an actual configuration.